sdk/storage/azure_storage_blob/src/models/extensions.rs

```diff
@@ -67,29 +68,18 @@ impl BlockBlobClientUploadOptions<'_> {
 }
 
 /// Converts a `BlobTags` struct into `HashMap<String, String>`.
-impl TryFrom<BlobTags> for HashMap<String, String> {
-    type Error = azure_core::Error;
-
-    fn try_from(blob_tags: BlobTags) -> Result<Self, azure_core::Error> {
+impl From<BlobTags> for HashMap<String, String> {
```

Comment:
Need to mark this down as breaking change? Mostly for the change from `try_into` to `into`.